SmartStore.podspec

```diff
@@ -22,7 +22,7 @@ Pod::Spec.new do |s|
       smartstore.dependency 'SalesforceSDKCore'
       smartstore.dependency 'FMDB/SQLCipher', '~> 2.7.5'
       smartstore.dependency 'SQLCipher/fts', '~> 4.2.0'
-      smartstore.source_files = 'libs/SmartStore/SmartStore/Classes/**/*.{h,m}', 'libs/SmartStore/SmartStore/SmartStore.h'
+      smartstore.source_files = 'libs/SmartStore/SmartStore/Classes/**/*.{h,m}', 'libs/SmartStore/SmartStore/SmartStore.h','libs/SmartStore/SmartStore/Classes/**/*.swift'
```

Comment:
Should we tweak our podspec generator script to include `swift` files?

Reply:
`update_podspec_headers.sh` updates the list of public headers, the source files never changes: it's a list of `**/*.h`, `**/*.m` and `**/*.swift`.